sklearn/discriminant_analysis.py

```diff
@@ -645,7 +657,13 @@ def fit(self, X, y):
             self.priors_ = self.priors
 
         cov = None
-        if self.store_covariances:
+        store_covariance = None
+        if self.store_covariances is not None:
+            warnings.warn("'store_covariances' was renamed to store_covariance"
+                          " in version 0.19 and will be removed in 0.21.",
+                          DeprecationWarning)
+            store_covariance = self.store_covariances
```

Comment:
but we can't figure out the value of `store_covariance ` is come from `self.store_covariance` or `self.store_covariances`, then we can't decide whether to raise a warning?